packages/components/src/checkbox-control/index.tsx

```diff
@@ -40,7 +67,7 @@ export default function CheckboxControl( {
 
 	// Run the following callback everytime the `ref` (and the additional
```

Comment:
```suggestion
	// Run the following callback every time the `ref` (and the additional
```

Small pre-existing typo.